apps/web/src/features/safenet/components/SafenetTxStatusSteps/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { StepState, TxSignerStep } from '@/components/transactions/TxSigners'
+import useIsSafenetEnabled from '../../hooks/useIsSafenetEnabled'
+import { useGetSafenetTransactionDetailsQuery } from '@/store/safenet'
+import useChainId from '@/hooks/useChainId'
+import { skipToken } from '@reduxjs/toolkit/query'
+import DotIcon from '@/public/images/common/dot.svg'
+import CheckIcon from '@/public/images/common/circle-check.svg'
+import CircleIcon from '@/public/images/common/circle.svg'
+
+import { Box, CircularProgress, Stack, SvgIcon, Typography } from '@mui/material'
+import css from './styles.module.css'
+
+const Dot = () => <SvgIcon component={DotIcon} inheritViewBox className={css.dot} />
+const Check = () => (
+  <SvgIcon
+    component={CheckIcon}
+    inheritViewBox
+    className={css.icon}
+    sx={{
+      '& path:last-of-type': { fill: ({ palette }) => palette.background.paper },
+    }}
+  />
+)
+
+const Incomplete = () => <SvgIcon component={CircleIcon} inheritViewBox className={css.icon} />
+
+export const SafenetTxStatusSteps = ({ safeTxHash }: { safeTxHash: string }) => {
```

Comment:
This should be exported with a dynamic import, like the `SafenetTransactionDetails` component